test/unittests/CMakeLists.txt

```diff
@@ -285,6 +285,7 @@ link_directories("/opt/lib/")
 # flags
 # ------------------------------------------------------------
 add_definitions(-DUNIT_TEST)
+add_definitions(-g)
```

Comment:
Was this necessary?
I've been running unitTest under gdb for a long long time without problems ... strange ...